sky/authentication.py

```diff
@@ -88,63 +84,20 @@ def get_or_generate_keys(private_key_path: str, public_key_path: str):
     return private_key, public_key
```

Comment:
nit: should we change the return type for `get_or_generate_keys` to return the paths to the keys instead of the keys themselves?

It seems both GCP and Azure auth need paths to the keys (L138, L139, L293, L296). We can add another fn to read the keys and invoke it when required:

```
def read_key(key_path):
    return open(key_path, 'rb').read().decode('utf-8')
```

Just a suggestion, feel free to ignore

Reply:
Good point! Fixed. Thanks!